spec/index.bs

```diff
@@ -793,17 +856,33 @@ has components working on different timelines in parallel:
     of view of the user agent part that controls the GPU,
     but can live in a separate OS process.
 
-    <div class=device-timeline>
-        Steps executed on the device timeline look like this.
+    <div class=algorithm>
+        This is an algorithm.
+
+        <div class=device-timeline>
+            Steps executed on the device timeline look like this.
+        </div>
+
+        <ol data-timeline=device>
+            1. Steps executed on the device timeline look like this.
+        </ol>
```

Comment:
Colorblind hat on: The color difference between the device and content timeline borders is a little hard for me to distinguish at a glance, though I can get it if I look at it for a moment. (Both are easily distinguished from the queue timeline color for me.)

I don't suspect that will be a big problem in practice because of the helpful "timeline" labels that get injected into every section, so it's only really a potential point of confusion if I'm reading a long, long algorithm. (And even then it's usually apparent where the code SHOULD be executing.)

Still, if we wanted to make it better I'd suggest slightly more saturated colors or a different color pick altogether. Maybe shifting the green towards yellow a bit more? Also making the left hand border a smidge wider so there's more of the color to see would help.